bandit/formatters/html.py

```diff
@@ -348,6 +349,9 @@ def report(manager, fileobj, sev_level, conf_level, lines=-1):
             candidates = candidate_block.format(candidate_list=candidates_str)
 
         url = docs_utils.get_url(issue.test_id)
+
+        print(issue)
```

Comment:
We probably don't want this

Reply:
Ahh crap. Sorry about that, I mistakenly left this in there while I was playing around. Will update.